app/views/ReportView/components/GenomicSummary/components/VariantChips/index.jsx

```diff
@@ -40,21 +40,62 @@ const VariantChips = (props) => {
     setAddedVariant('');
   }, [onChipAdded, addedVariant]);
 
-  const handleEnterPressed = (event) => {
+  const handleEnterPressed = useCallback((event) => {
     if (event.keyCode === ENTER_KEYCODE) {
       handleAddInputClose(true);
     }
+  }, [handleAddInputClose]);
+
+  let newGeneAlterationButton = null;
+  if (canEdit && !isPrint) {
+    if (!showAddInput) {
+      newGeneAlterationButton = (
+        <Chip
+          label="Add new entry"
+          className="variant variant__add"
+          onDelete={() => setShowAddInput(true)}
+          deleteIcon={<AddIcon />}
+        />
+      );
+    } else {
+      newGeneAlterationButton = (
+        <TextField
+          value={addedVariant}
+          type="text"
+          size="small"
+          margin="dense"
+          variant="outlined"
+          label="Gene name (alteration)"
+          onKeyDown={handleEnterPressed}
+          onChange={(event) => setAddedVariant(event.target.value)}
+          InputProps={{
+            endAdornment: (
+              <InputAdornment position="end">
+                {addedVariant && (
+                  <IconButton onClick={() => handleAddInputClose(true)} size="large">
+                    <DoneIcon />
+                  </IconButton>
+                )}
+                <IconButton onClick={() => handleAddInputClose(false)} size="large">
+                  <CloseIcon />
+                </IconButton>
+              </InputAdornment>
+            ),
+          }}
+        />
+      );
+    }
   }
 
   return (
     <div>
       <div>
         {Boolean(variants.length) && (
           <>
-            {variants.map(variant => (
+            {variants.map((variant) => (
               <React.Fragment key={variant.geneVariant}>
                 <Chip
-                  label={variant.geneVariant}
+                  label={`${variant.geneVariant} ${variant.germline ? '(germline)' : ''}`}
```

Comment:
Just this line is the thing, everything else is just moving things around

Reply:
can skip for this one, but in future it would be helpful if changes that were just refactoring were split into their own commit